src/Sylius/Behat/Page/Admin/TaxCategory/CreatePage.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page\Admin\TaxCategory;
+
+use Sylius\Behat\Behaviour\DescribesItAs;
+use Sylius\Behat\Behaviour\NamesIt;
+use Sylius\Behat\Behaviour\SpecifiesCode;
+use Sylius\Behat\Page\Admin\Crud\CreatePage as BaseCreatePage;
+
+/**
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ */
+class CreatePage extends BaseCreatePage implements CreatePageInterface
+{
+    use SpecifiesCode;
+    use NamesIt;
+    use DescribesItAs;
```

Comment:
Let's be consistent:
* Specifies**Its**Code :question: 
* NamesIt :ok: 
* DescribesIt~~**As**~~